services/horizon/internal/db2/schema/migrations/43_asset_stat_trust_line_flags.sql

```diff
@@ -0,0 +1,16 @@
+-- +migrate Up
+
+ALTER TABLE asset_stats
+ADD COLUMN accounts JSONB;
+ADD COLUMN balances JSONB;
+-- Previously, all the asset_stats we stored were authorized.
+UPDATE asset_stats
+  SET
+    accounts = jsonb_build_object('authorized', num_accounts),
+    balances = jsonb_build_object('authorized', amount);
```

Comment:
Shouldn't this be `json_build_object` instead of `jsonb_build_object`?

```suggestion
    accounts = json_build_object('authorized', num_accounts),
    balances = json_build_object('authorized', amount);
```

Reply:
No, json and jsonb are different (but similar) things in postgres.

Reply:
Got it, thanks for explaining.